python/tests/pytest.json

```diff
@@ -0,0 +1,32 @@
+[
+    {
+        "data_func": "constant_function",
+        "data_func_args": [2000,5,1,100],
+        "assert_func": "assert_prediction",
+        "assert_func_args": [[5, 0],1],
+        "grid":   {
+            "#base": ["-P 50000 --preserve_performance_counters --save_resume "],
+            "#reg": ["", "--coin"]
+        }
+    },
+    {
+        "data_func": "constant_function",
+        "data_func_args": [2000,5,1,100],
+        "assert_func": "assert_weight",
+        "assert_func_args": [[5, 0],1],
+        "grid":   {
+            "#base": ["-P 50000 --preserve_performance_counters --save_resume "],
+            "#reg": ["", "--coin"]
+        }
+    },
+    {
+        "data_func": "constant_function",
+        "data_func_args": [2000,5,1,100],
+        "assert_func": "assert_weight",
+        "assert_func_args": [[5, 0],1],
+        "grid":   {
```

Comment:
learning_rate is specific argument for sgd ("") - others learners have another hyperparmeters (i.e. ftrl has ftrl_alpha and ftrl_beta).

In python there is a way to use '+'/'*' operator over grids in the following way:
```
grid = Grid(
    {
        '#base': '-P 1000 ...'
    }) * (
        Grid({
            "#lr": "",
            "--learning_rate": [0.01, 0.1, 1]
            }) + Grid({
                '#lr': '--ftrl',
                '--ftrl_alpha': [0.01, 0.1, 1]
                }))
```
Would be good to consider being able to express that kind of things in the future.

Reply:
this is just to consider for the future